tls/s2n_psk.h

```diff
@@ -80,6 +80,12 @@ int s2n_psk_free(struct s2n_psk *psk);
 S2N_RESULT s2n_psk_parameters_init(struct s2n_psk_parameters *params);
 S2N_CLEANUP_RESULT s2n_psk_parameters_wipe(struct s2n_psk_parameters *params);
 
+struct s2n_psk_identity_list {
+    struct s2n_stuffer *identities;
+};
+S2N_RESULT s2n_psk_identity_list_get(struct s2n_psk_identity_list *list, uint16_t index,
+        uint8_t **output, uint16_t *output_size);
```

Comment:
Why is the index `uint16_t`, we could use `size_t` instead right?

Reply:
We could, but the index can never exceed uint16_t, so it would be unnecessary.